rebar.config

```diff
@@ -49,7 +49,7 @@
 {swagger_endpoints, [{src, "config/swagger.yaml"}, {dst, "apps/aeutils/src/endpoints.erl"}]}.
 
 {relx, [{release, { epoch, "version value comes from VERSION" },
-         [runtime_tools, lager, setup, sext, rocksdb, mnesia_rocksdb, gproc,
+         [runtime_tools, sasl, lager, setup, sext, rocksdb, mnesia_rocksdb, gproc,
```

Comment:
Could you please add small comment in PR on why and expected impact?

Reply:
Thinking more about this - could you please split sasl-changes in separate PR? Reason is that I would like to avoid possibility of creating unexpected volume or format of logs close to stabilization of master.

Reply:
Adding SASL was to fix the command that wasn't working in the first place. I understand that you may want to wait before merging but the PR doesn't make sense without that.

Reply:
Which command?
If @dincho is ok to include sasl (for amount and format of logs), I am ok.

Reply:
Lager should take over SASL events. Including SASL in a production system is very good practice IMO.

Reply:
`epoch versions` is not working without sasl.